src/UI/TokenDetails/TokenSymbol/index.tsx

```diff
@@ -1,23 +1,31 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
 import classNames from 'classnames';
-import styles from 'UI/TokenDetails/TokenSymbol/tokenSymbol.scss';
+
 import { WithClassnameType } from '../../types';
 
+import styles from 'UI/TokenDetails/TokenSymbol/tokenSymbol.scss';
+
+export interface SimplePropsType extends WithClassnameType {
+  children: ReactNode;
+}
+
+export interface CombinedPropsType extends WithClassnameType {
+  small: boolean | undefined;
+  children: ReactNode;
+}
+
 export const Simple = ({
   children,
   className = 'dapp-simple-token-symbol'
-}: { children: React.ReactNode } & WithClassnameType) => (
+}: SimplePropsType) => (
   <div className={classNames(styles.tokenSymbol, className)}>{children}</div>
 );
 
 export const Combined = ({
```

Comment:
add TODO to change naming into something more relevant in v3

Reply:
Added. 👍